### ISSUE TITLE
selenium/chromedriver method not working on OSX

### ISSUE THREAD

**Issue by dzg** [mrooney/mintapi#100]
Trying to get this to work on OSX Sierra 10.12.4.

I installed all the packages successfully.

From python2, I did `mint = mintapi.Mint(email, password)`

This launched a new instance of Chrome opening the Mint website, where I tried to log in.

But then I got this in python console:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/mintapi/api.py", line 69, in __init__
    self.login_and_get_token(email, password, ius_session, thx_guid)
  File "/usr/local/lib/python2.7/site-packages/mintapi/api.py", line 135, in login_and_get_token
    session_cookies = self.get_session_cookies(**data)
  File "/usr/local/lib/python2.7/site-packages/mintapi/api.py", line 187, in get_session_cookies
    driver.find_element_by_link_text("Log In").click()
  File "/usr/local/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 337, in find_element_by_link_text
    return self.find_element(by=By.LINK_TEXT, value=link_text)
  File "/usr/local/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 791, in find_element
    'value': value})['value']
  File "/usr/local/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 256, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"link text","selector":"Log In"}
  (Session info: chrome=58.0.3029.110)
  (Driver info: chromedriver=2.29.461585 (0be2cd95f834e9ee7c46bcc7cf405b483f5ae83b),platform=Mac OS X 10.12.4 x86_64)
```

I tried the `chrome://settings/cookies` method, and found`ius_session` and `thx_guid` --  in different cookies -- `pf.intuit.com` and `accounts.intuit.com` -- and this worked!

Thanks

**Comment by mrooney** [comment]
Glad that worked! Closing as this appears to be a duplicate of #95

**Comment by rafilurie** [comment]
@mrooney can you explain the fix? I don't understand "I tried the chrome://settings/cookies method, and foundius_session and thx_guid -- in different cookies -- pf.intuit.com and accounts.intuit.com -- and this worked!"

I am currently getting the following error based on the 1.39 update: 

`File "/Users/rafilurie/Projects/cash/flaskvue/backend/venv/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/rafilurie/Projects/cash/flaskvue/backend/venv/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"link text","selector":"LOG IN"}
  (Session info: chrome=75.0.3770.142)
  (Driver info: chromedriver=2.41.578706 (5f725d1b4f0a4acbf5259df887244095596231db),platform=Mac OS X 10.14.2 x86_64)`

**Comment by mrooney** [comment]
Can you try the just-released 1.40? I don't think 1.39 was released properly with the fix.
